### PR TITLE
Removed dead code per issue 8677 (https://github.com/OpenRCT2/OpenRCT2/issues/8677)

### PR DESCRIPTION
**Pull request by phillipwalters**

### CODE REVIEW

**Comment by AaronVanGeffen** [comment]
Could you elaborate on how you've verified that this is indeed dead code, as suspected?

File: src/openrct2/Game.cpp [hunk 1]
```diff
@@ -630,96 +615,6 @@ void game_log_multiplayer_command(int command, const int* eax, const int* ebx, c
         format_string(log_msg, 256, STR_LOG_DEMOLISH_RIDE, args);
         network_append_server_log(log_msg);
     }
-    else if (
-        command == GAME_COMMAND_SET_RIDE_APPEARANCE || command == GAME_COMMAND_SET_RIDE_VEHICLES
```
Comment (ZehMatt) [inline src/openrct2/Game.cpp]: This is not dead code. GAME_COMMAND_SET_RIDE_VEHICLES is still used and GAME_COMMAND_SET_RIDE_SETTING

File: src/openrct2/Game.cpp [hunk 2]
```diff
@@ -773,9 +652,7 @@ void game_log_multiplayer_command(int command, const int* eax, const int* ebx, c
         format_string(log_msg, 256, STR_LOG_PLACE_SCENERY, args);
         network_append_server_log(log_msg);
     }
-    else if (
-        command == GAME_COMMAND_REMOVE_SCENERY || command == GAME_COMMAND_REMOVE_WALL
-        || command == GAME_COMMAND_REMOVE_LARGE_SCENERY || command == GAME_COMMAND_REMOVE_BANNER)
```
Comment (ZehMatt) [inline src/openrct2/Game.cpp]: GAME_COMMAND_REMOVE_BANNER is still in use.

File: src/openrct2/Game.cpp [hunk 3]
```diff
@@ -794,8 +671,7 @@ void game_log_multiplayer_command(int command, const int* eax, const int* ebx, c
     else if (
         command == GAME_COMMAND_SET_SCENERY_COLOUR || command == GAME_COMMAND_SET_WALL_COLOUR
         || command == GAME_COMMAND_SET_LARGE_SCENERY_COLOUR || command == GAME_COMMAND_SET_BANNER_COLOUR
-        || command == GAME_COMMAND_SET_BANNER_NAME || command == GAME_COMMAND_SET_SIGN_NAME
-        || command == GAME_COMMAND_SET_BANNER_STYLE || command == GAME_COMMAND_SET_SIGN_STYLE)
```
Comment (ZehMatt) [inline src/openrct2/Game.cpp]: GAME_COMMAND_SET_BANNER_STYLE  is still in use.

**Comment by phillipwalters** [comment]
> Could you elaborate on how you've verified that this is indeed dead code, as suspected?

I asked in the bug report if there was any way to verify and was told to just send the pull request. Apologies!

**Comment by AaronVanGeffen** [comment]
> I asked in the bug report if there was any way to verify and was told to just send the pull request. Apologies!

Ah, no worries then! I missed that bit.

File: src/openrct2/Game.cpp [hunk 4]
```diff
@@ -794,8 +671,7 @@ void game_log_multiplayer_command(int command, const int* eax, const int* ebx, c
...
+        || command == GAME_COMMAND_SET_SIGN_NAME || command == GAME_COMMAND_SET_BANNER_STYLE)
```
Comment (duncanspumpkin) [inline src/openrct2/Game.cpp]: Sign name is game action and can therefore be removed

File: src/openrct2/Game.cpp (hunk 1 above)
Comment (phillipwalters) [reply src/openrct2/Game.cpp]: Sorry about that. I've put it back in.

File: src/openrct2/Game.cpp (hunk 4 above)
Comment (phillipwalters) [reply src/openrct2/Game.cpp]: Removed!

**Comment by duncanspumpkin** [comment]
You just need to make a small adjustment to the formatting to keep the clang format ci happy. https://travis-ci.org/OpenRCT2/OpenRCT2/builds/491956701

File: src/openrct2/Game.cpp (hunk 1 above)
Comment (ZehMatt) [reply src/openrct2/Game.cpp]: No worries, your PR is appreciated.

**Comment by phillipwalters** [comment]
> You just need to make a small adjustment to the formatting to keep the clang format ci happy. https://travis-ci.org/OpenRCT2/OpenRCT2/builds/491956701

Changes made. Both Travis and AppVeyor  builds are now passing.

**Pull request merged**